common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -2082,8 +2082,13 @@
   "screens.wynntils.wynntilsContentBook.sort.distance.name": "Sort by Distance",
   "screens.wynntils.wynntilsContentBook.sort.level.description": "Quests are sorted by level, lowest first",
   "screens.wynntils.wynntilsContentBook.sort.level.name": "Sort by Level",
+  "screens.wynntils.wynntilsCrowdSourcing.collecting": "Collecting",
+  "screens.wynntils.wynntilsCrowdSourcing.copiedToClipboard": "Copied %d %s to clipboard",
```

```suggestion
  "screens.wynntils.wynntilsCrowdSourcing.copiedToClipboard": "Copied %d data points of %s to clipboard",
```

Maybe this is better?